src/views/product-integration/component-view/components/variable-group-list/index.tsx

```diff
@@ -56,6 +56,8 @@ export interface VariableGroupListProps {
 	isNested?: boolean
 }
 
+/** @TODO refactor this to use the `Card` component within the list item */
```

Comment:
We wouldn't need to add the override below if this component was using `Card`. That felt a bit out of scope for this PR but I created a follow up ticket.